packages/gatsby-source-wordpress/README.md

```diff
@@ -83,6 +83,8 @@ plugins: [
       // Set verboseOutput to true to display a verbose output on `npm run develop` or `npm run build`
       // It can help you debug specific API Endpoints problems
       verboseOutput: false,
+      // Search and Replace Urls across WordPress content
+      searchReplace: ["https://source-url.com", "https://replacement-url.com"]
```

Comment:
Could we change this to use named fields e.g.

```javascript
searchReplace: {
  sourceUrl: `https://source-url.com`,
  replacementUrl: `https://replacement-url.com`,
},
```

I find array-based APIs really ambiguous as the position of an item in an array isn't much of a cue to how you should use it or later, what it even means.

Reply:
Search & Replace option has been updated to accept an object rather than an array